quicksort/QuickSort.java

```diff
@@ -0,0 +1,36 @@
+public class QuickSort {
```

Comment:
That would have fixed this error. But doesn't matter, now I will remove this check from codacy.